### PR TITLE
Update the install guide with the addition of installing via `krew`

### PR DESCRIPTION
**Pull request by dhiller**

### CODE REVIEW

File: administration/intro.adoc [hunk 1]
```diff
@@ -155,8 +155,29 @@ It also provides convenience commands for:
...
+$ kubectl krew install virt
+----
+
+Then `virtctl` can be used as a kubectl plugin. For a list of available commands run:
+
+[source,bash]
+----
+$ kubectl virt help
+----
+
```
Comment (fabiand) [inline administration/intro.adoc]: Please  note that all appearances of `virtctl` throughout this guide should be read as a call to `krew virt` if krew is used.

Comment (dhiller) [reply administration/intro.adoc]: @fabiand Done.

**Comment by dhiller** [comment]
@fabiand please review again.

**Comment by fabiand** [comment]
Thanks!

**Pull request merged**